AzureMonitorAgent/agent.py

```diff
@@ -421,23 +421,41 @@ def install():
             data = []
             new_data = ""
             vars_set = set()
+            dependent_vars = set(default_configs.keys()).intersection(["MDSD_OPTIONS"])
+
+            # Copy existing comments/envvar to the updated defaults file; replace existing envvar values if appropriate
             with open(config_file, "r") as f:
```

Comment:
Separate variable for mdsd options array

Reply:
@simathih fixed, thanks